### PR TITLE
Fixed hard-coded comrade numbers and temporarily disabled some buttons

### PR DESCRIPTION
**Pull request by HenryDang98**
Comrade numbers are no longer hard-coded
Comrade numbers are now saved and editable after being submitted

Buttons that lead to blank activities are temporarily commented out until the functionality for the associated activity is implemented.

### CODE REVIEW

File: app/src/main/java/com/peacecorps/pcsa/MainActivityFragment.java [hunk 1]
```diff
@@ -39,14 +41,18 @@ public void onClick(View v) {
         getHelpNowButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                startActivity(new Intent(getActivity(), HomeScreen.class));
+                //Temporarily commented out until the functionality is implemented.
+                //startActivity(new Intent(getActivity(), HomeScreen.class));
+                Toast.makeText(getActivity(), "This functionality is currently unavailable.", Toast.LENGTH_SHORT).show();
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/MainActivityFragment.java]: Move the string to the string.xml file

**Comment by sandarumk** [comment]
Great work! This PR has only one mistake. You have to move the string in the toast message to string.xml file. Otherwise this is ready to merge. Apologies from my side that I haven't noticed that before.

**Comment by HenryDang98** [comment]
@sandarumk 

I've changed it now so that they're in the String.xml file.

File: app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java [hunk 2]
```diff
@@ -46,12 +49,27 @@ protected void onCreate(Bundle savedInstanceState) {
...
-                Toast.makeText(getApplicationContext(),"Comrades phone numbers has been updated", Toast.LENGTH_LONG).show();
+                editor.putString(comrade1, comrade1editText.getText().toString());
+                editor.putString(comrade2, comrade2editText.getText().toString());
+                editor.putString(comrade3, comrade3editText.getText().toString());
+                editor.putString(comrade4, comrade4editText.getText().toString());
+                editor.putString(comrade5, comrade5editText.getText().toString());
+                editor.putString(comrade6, comrade6editText.getText().toString());
+
+                editor.commit();
+                Toast.makeText(getApplicationContext(), "Comrades phone numbers has been updated", Toast.LENGTH_LONG).show();
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java]: Please add this string to the string.xml as well. The standard is you don't use any hard coded strings in the code itself. You add all the strings to string.xml and refer from there.

**Comment by HenryDang98** [comment]
Okay, I've moved every hard-coded string into strings.xml.

**Comment by sandarumk** [comment]
@rosariorobinson This PR is ready to merge.

**Pull request merged**